man/man7/vdevprops.7

```diff
@@ -104,12 +104,19 @@ Comma separated list of children of this vdev
 The number of children belonging to this vdev
 .It Sy read_errors , write_errors , checksum_errors , initialize_errors , trim_errors
 The number of errors of each type encountered by this vdev
+.It Sy sit_out_reads
+True when a slow disk outlier was detected and the vdev is currently in a sit
+out state.
+While sitting out, the vdev will not participate in normal reads, instead its
+data will be reconstructed as needed from parity.
 .It Sy slow_ios
 The number of slow I/Os encountered by this vdev,
 These represent I/O operations that didn't complete in
 .Sy zio_slow_io_ms
 milliseconds
 .Pq Sy 30000 No by default .
+Can also be incremented when a vdev was determined to be a raidz leaf latency
+outlier.
```

Comment:
I'm not sure I understand what you mean by this.  Who would be incrementing `zio_slow_io_ms`?

Reply:
how about this...
```
     slow_ios       This indicates the number of slow I/O operations encountered
                    by this vdev.  A slow I/O is defined as an operation that
                    did not complete within the zio_slow_io_ms threshold in mil‐
                    liseconds (30000 by default).  For RAIDZ and DRAID configu‐
                    rations, this value also represents the number of times the
                    vdev was identified as an outlier and excluded from partici‐
                    pating in read I/O operations.
```